index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Restaurant Fiescherblick</title>
+    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <!-- If you're using google fonts, that should be added here -->
+  </head>
+ 
+  <body>
+    <div id="home" class="header">
+      <a href="./index.html"class="logo"><img src="icons/Fiescherblick Logo grün.svg" alt="Logo Fiescherblick"></a>
```

Comment:
Really nice that you put your images and icons in different folders. It makes it so much easier to overview your project ⭐